migrations/scripts/1_unrouted.js

```diff
@@ -0,0 +1,149 @@
+const fs = require("fs")
+const { merge } = require("lodash")
+const utils = require("../../scripts/utils")
+
+const addresses = require("../addresses")
+const requests = require("../witnet.requests")
+const settings = require("../settings")
+
+module.exports = async function (deployer, network) {
+  const rn = require("../../scripts/utils").getRealmNetworkFromString(network.split("-")[0])
```

Comment:
Why can't use `utils` imported in [line 3](https://github.com/witnet/witnet-price-feed-examples/pull/94/files#diff-559cdfb830a470338b4ff79bde78fc6f11c62d626ae8c9b81a553f7184e97495R3)?